mods/d2k/sequences/structures.yaml

```diff
@@ -35,9 +35,9 @@ guntower:
 		Facings: -32
 		Offset: -24,16
 	muzzle: DATA.R8
-		Frames: 3839, 3839, 3840, 3840, 3841, 3841, 3842, 3842, 3843, 3843, 3844, 3844, 3845, 3845, 3846, 3846, 3847, 3847, 3848, 3848, 3849, 3849, 3850, 3850, 3851, 3851, 3852, 3852, 3853, 3853, 3854, 3854, 3855, 3855, 3856, 3856, 3857, 3857, 3858, 3858, 3859, 3859, 3860, 3860, 3861, 3861, 3862, 3862, 3863, 3863, 3864, 3864, 3865, 3865, 3866, 3866, 3867, 3867, 3868, 3868, 3869, 3869, 3870, 3870
+		Start: 3839
+		Tick: 60
 		Facings: -32
-		Length: 2
```

Comment:
As for the reason why the gun tower needs the trike muzzle, it's impossible to get the offset right for the tank cannon muzzle because the offset baked into the tank muzzle .r8, the offset baked into the turret art .r8 and the firing offset in the yaml rule don't like each other.